pkg/identityserver/gateway_registry.go

```diff
@@ -283,6 +328,21 @@ func (is *IdentityServer) updateGateway(ctx context.Context, req *ttnpb.UpdateGa
 			return nil, err
 		}
 	}
+	if ttnpb.HasAnyField(req.FieldMask.Paths, "lbs_lns_secret") && req.LBSLNSSecret != nil {
```

Comment:
Why that `req.LBSLNSSecret != nil`? Is it not allowed to unset this field?

Reply:
should we allow that?